### PR TITLE
Clarify auction info

### PR DESCRIPTION
**Pull request by DrW3RK**
Minor fixes after the PR https://github.com/w3f/polkadot-wiki/pull/3026

### CODE REVIEW

File: docs/learn/learn-auction.md [hunk 1]
```diff
@@ -31,63 +31,69 @@ extinguished and the candle went out, the auction would suddenly terminate and t
...
-to end based on the candle auction mechanism. 
-
-The auction’s ending time can be any time within this ending period, and is automatically and randomly 
-chosen by the [Verifiable Random Function (VRF)](learn-randomness.md##vrf). The probability of winning 
-the auction is equal to the number of blocks with a winning bid divided by the total number of blocks in 
-the ending period. The random ending is managed by propagating through the entire ending period, where a 
-snapshot is taken at each block within the ending period to capture the winners during that block. At the 
-end of the period, one of the snapshots is randomly selected to determine the winner of the auction.
-
-This process is executes six hours right after the ending period.
```
Comment (profradha) [inline docs/learn/learn-auction.md:53]: Further explained this sentence "This process is executes six hours right after the ending period."

by rephrasing it to - "This process is executed in the next epoch (which lasts for six hours on Polkadot) after the ending period."

File: docs/learn/learn-auction.md [hunk 2]
```diff
@@ -31,63 +31,69 @@ extinguished and the candle went out, the auction would suddenly terminate and t
 that point would win.
 
 When candle auctions are used online, they require a random number to decide the moment of
-termination. Parachain slot auctions differ slightly from a normal candle auction in that it does not
```
Comment (profradha) [inline docs/learn/learn-auction.md:34]: Rephrased this sentence which is factually incorrect, as a random number is never used to determine the opening phase - "Parachain slot auctions differ slightly from a normal candle auction in that it does not use the random number to decide the duration of its opening phase." 

to - "Parachain slot auctions differ slightly from a normal candle auction in that they do
not randomly terminate the auction. Instead, they run for an entire fixed duration and the winner is
chosen retroactively."

File: docs/learn/learn-auction.md [hunk 3]
```diff
@@ -31,63 +31,69 @@ extinguished and the candle went out, the auction would suddenly terminate and t
...
+
+The auction’s ending time can be any time within this ending period, and is automatically and
+randomly chosen by the [Verifiable Random Function (VRF)](learn-randomness.md##vrf). The probability
+of winning the auction is equal to the number of blocks with a winning bid divided by the total
+number of blocks in the ending period. The random ending is managed by propagating through the
+entire ending period, where a snapshot is taken at each block within the ending period to capture
+the winners during that block. At the end of the period, one of the snapshots is randomly selected
+to determine the winner of the auction.
+
+This process is executed in the next epoch (which lasts for six hours on Polkadot) after the ending
```
Comment (salmad3) [inline docs/learn/learn-auction.md]:
```suggestion
This process executes in the next epoch (which lasts for six hours on Polkadot) after the ending
```

File: docs/learn/learn-auction.md [hunk 4]
```diff
@@ -31,63 +31,69 @@ extinguished and the candle went out, the auction would suddenly terminate and t
...
-> these six hours when the winning block for the auction is being determined on-chain.
-  
-With one day and eighteen hours for the starting period, five days for the ending period (candle auction 
-phase) and six hours for determining the auction winner, a parachain auction on Polkadot lasts exactly one week 
-from the start.
-
-More details on this is available in the [Polkadot Implementation](#polkadot-implementation) section.
+termination. Parachain slot auctions differ slightly from a normal candle auction in that they do
+not randomly terminate the auction. Instead, they run for an entire fixed duration and the winner is
+chosen retroactively.
```
Comment (profradha) [inline docs/learn/learn-auction.md]:
```suggestion
randomly chosen retroactively.
```

**Review by salmad3 — COMMENTED** [review]
Thanks @DrW3RK just a minor suggestion

**Comment by DrW3RK** [comment]
The prettify command is messing with bold formatting, changing it to italic. How do you get around that?

**Comment by salmad3** [comment]
> The prettify command is messing with bold formatting, changing it to italic. How do you get around that?

doing it manually

**Review by salmad3 — APPROVED** [review]

**Pull request merged**